src/AzureExtension/DevBox/DevBoxInstance.cs

```diff
@@ -484,6 +487,12 @@ public IAsyncOperation<IEnumerable<ComputeSystemProperty>> GetComputeSystemPrope
         }).AsAsyncOperation();
     }
 
+    public IApplyConfigurationOperation CreateApplyConfigurationOperation(string configuration)
+    {
+        var taskAPI = $"{DevBoxState.Uri}{Constants.CustomizationAPI}{DateTime.Now.ToFileTimeUtc()}?{Constants.APIVersion}";
+        return new ConfigWrapper(configuration, taskAPI, _devBoxManagementService, AssociatedDeveloperId, _log);

```

Comment:
I feel we do otherwise we will have to pass more details about the dev box to initialize, so it will be the same effect.

For the side note, I agree. Changed.